1-JavaScript/Week2/assignment/ex1-giveCompliment.js

```diff
@@ -17,13 +17,26 @@ Full description at: https://github.com/HackYourFuture/Assignments/tree/main/1-J
    Use `console.log` each time to display the return value of the
    `giveCompliment` function to the console.
 -----------------------------------------------------------------------------*/
-export function giveCompliment(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+export function giveCompliment(name) {
+  const compliments = [
+    'great',
+    'awesome',
+    'amazing',
+    'hero',
+    'creative',
+    'genius',
+    'brilliant',
+    'talented',
+    'exceptional',
+    'inspiring',
+  ];
+  const randomNumber = Math.floor(Math.random() * 10);
```

Comment:
```suggestion
  const randomNumber = Math.floor(Math.random() * compliments.length);
```

Suggestion: 
Instead of hardcoding the number 10, it's better to use the array's length for more maintainable code.